### ISSUE TITLE
Mac Mouse Fix prevents me from using middle click in apps such as Sketchup and Archicad to pan, rotate and orbit in 3d. [other]

### ISSUE THREAD

**Issue by angozi** [noah-nuebling/Mac-Mouse-Fix#621]
I would like on option to disable Mac mouse fix on specific apps, or a none/default option in the middle click settings panel

**Comment by ghost** [comment]
Yes, this. I encountered the same thing in 3D programs. Mac Mouse Fix needs a way to do "passthrough" on particular actions. Personally I _only_ want to use it to customize side buttons. I don't need any customization for middle mouse.

This problem is so severe that it will make me disable Mac Mouse Fix entirely.

**Comment by noah-nuebling** [comment]
Hi there, thanks for reaching out.

There's a section "How can I orbit around objects in 3D apps like Blender?" in the new [Readme](https://github.com/noah-nuebling/mac-mouse-fix). Does that help?

**Comment by psynchro303** [comment]
That part in the readme resolved my issue, thanks very much.
